parser/formattext.go

```diff
@@ -133,6 +136,14 @@ func (fc *FontConfig) FormatText(text string, maxWidth int, cursorOverlapWidth i
 	return formattedSb.String(), nil
 }
 
+func (fc *FontConfig) isFirstLine(lineNumber int) bool {
+   return lineNumber == 1
+}
+
+func (fc *FontConfig) isMaxLineOrGreater(lineNumber int, NumLines int) bool {
```

Comment:
This function should be renamed to `shouldUseLineFeed()`.

Reply:
https://github.com/huderlem/poryscript/pull/59/commits/adc1855b1c59625857cf38e0d402cf936c18a332